dev_docs/getting_started/hello_world_plugin.mdx

```diff
@@ -0,0 +1,144 @@
+---
+id: kibHelloWorldApp
+slug: /kibana-dev-docs/hello-world-app
+title: Hello World
+summary: Build a very basic plugin that registers an application that says "Hello World!".
+date: 2021-08-03
+tags: ['kibana', 'dev', 'contributor', 'tutorials']
+---
+
+This tutorial walks you through two ways to create a plugin that registers an application that says "Hello World!".
+
+You can view the tested example plugin at [examples/hello_world](https://github.com/elastic/kibana/tree/master/examples/hello_world).
+
+## 1. Set up your development environment
+
+Read through <DocLink id="kibDevTutorialSetupDevEnv" text="these instructions"/> to get your development environment set up.
+
+## 2. Option 1 - Write it manually
+
+This is a good option if you want to understand the bare minimum needed to register a "Hello world" application. The example plugin is based off of this option.
+
+1. Create your plugin folder. Start off in the `kibana` folder.
+
+```
+> cd examples
+> mkdir hello_world
+> cd hello_world
+```
+
+2. Create the kibana.json manifest file.
+
+```
+> echo '{
+  "id": "helloWorld",
+  "version": "1.0.0",
+  "kibanaVersion": "kibana",
+  "ui": true
+}' > kibana.json
+```
+
+3. Create a `tsconfig.json` file.
+
+```
+> echo '{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./target",
+    "skipLibCheck": true
+  },
+  "include": [
+    "index.ts",
+    "common/**/*.ts",
+    "public/**/*.ts",
+    "public/**/*.tsx",
+    "server/**/*.ts",
+    "../../typings/**/*"
+  ],
+  "exclude": []
+}
+' > tsconfig.json
+```
+
+4. Create a `public/plugin.tsx` file.
+
+```
+> mkdir public
+> touch plugin.tsx
+```
+
+Add the following to your `plugin.tsx` file:
```

Comment:
add link to plugin docs, possibly also some additional text quickly mentioning what this does.

Reply:
Linked to the doc that has more info. The current "Create a plugin" tutorial I'm going to change to "Anatomy of a plugin" and move it out of tutorials and into key concepts.